src/test/java/net/masterthought/cucumber/ReportBuilderTest.java

```diff
@@ -173,22 +166,6 @@ public void copyStaticResources_CopiesRequiredFiles() {
         assertThat(files).hasSize(21);
     }
 
-    @Test
-    public void createEmbeddingsDirectory_CreatesDirectory() {
```

Comment:
in the past I had a problem with this directory - did you remove this test forever or move it to some other class?